google/auth/_cloud_sdk.py

```diff
@@ -42,8 +44,11 @@
 )
 
 
-def get_config_path():
-    """Returns the absolute path the the Cloud SDK's configuration directory.
+def get_config_path(config_directory=_CONFIG_DIRECTORY):
```

Comment:
Maybe we should wrap this function with a `gcloud_get_config_path` so the intent is clearly documented. 

Would it be a large change to modify existing uses of `get_config_path`?

Reply:
Updated, not too much overhead to modify existing uses of `get_config_path`